.github/workflows/deploy-to-production.yml

```diff
@@ -3,7 +3,7 @@ on:
   release:
     types: [created]
 env:
-  terraform_version: '0.12.29'
+  terraform_version: '0.14.8'
```

Comment:
Nice, guess this plus the lockfile guarantees that things are updated together?

Reply:
Exactly :-D